README.md

```diff
@@ -27,6 +27,10 @@ centers and universities:
 
 The extensions have been evaluated in multiple academic publications such as:
 
+> Marco Solieri, Tomasz Kloda, Marko Bertogna, Michal Sojka, Maxim Baryshnikov,
+> "D5. 3: Integrated schedulability analysis",
+> Deliverable of the HERCULES project, 2018
```

Comment:
@soujak Please provide a direct link and a reference to the pages. All the other reference can be found easily with a simple search and are self contained. This one cannot be easily accessed.

Reply:
The project website disappeared, but we can provide the latest URL available from The Internet Archive.

Reply:
And also the academic repository provided by UNIMORE.